### ISSUE TITLE
Feature request: CalDAV and CardDAV support

### ISSUE THREAD

**Issue by enoversum** [nylas/N1#521]
Since I said good-bye to Google a long time ago, I would love to be able to sync at least my CardDAV contacts (i.e. always having the latest contacts, self-updating/-fetching).

**Comment by screendriver** [comment]
+1

**Comment by bengotow** [comment]
Hey folks! Long term, we want N1 to have rich contact integration, both with CardDAV services and with local sources, like Mac OS X's address book. It's a long ways off unfortunately, but pull requests are very welcome. The backend sync engine project doesn't integrate with CardDAV, so there's a lot of work to be done. If anyone knows of other open-source projects we could leverage for CardDAV integration, it'd be really helpful!

**Comment by enoversum** [comment]
Hey @bengotow , am not a developer of that sort, unfortunately, so having not a real clue as where to start. But thanks for letting me know. Just out of curiosity, N1 already detects a lot of my addresses, where do they come from (since no Gmail is in use)?

**Comment by ChristopheLux** [comment]
Hello @bengotow ,
I don't know if the licence is compatible but in open-source
https://www.inf-it.com/open-source/clients/carddavmate/ for CARDDav
and https://www.inf-it.com/open-source/clients/caldavzap/ for CALDAV

Hope that helps a bit

**Comment by tmlmt** [comment]
+1 ! CardDAV to quickly find the email address of our recipients, and CalDAV to handle event invitations.

**Comment by dominikzogg** [comment]
+1

**Comment by raph-ael** [comment]
a very BIG +1 for carddav AND caldav integration! 
and a much BIGGER thanks for this beutiful peace of open-source software! It looks like to replace my many years used thunderbird...

**Comment by mst-online** [comment]
+1

**Comment by zombxx** [comment]
:+1:

**Comment by NodokaMurmevent** [comment]
:+1: i use thunderbird with caldav + cardav with my private owncloud, i cannot leave him without this.

**Comment by ozsk** [comment]
:+1:

**Comment by spacemanspiff2007** [comment]
:+1:
Same here, owncloud with caldav

**Comment by avollmerhaus** [comment]
Without Caldav / Carddav, we can't switch. Backend is SOGo. At the moment we use Thunderbird, but it definitely has it's rough edges and there is no clear roadmap for the future...

**Comment by sysfu** [comment]
If you could integrate the fruux.com CardDAV/CalDAV service, which is the commercial arm of the [SabreDAV](https://github.com/fruux/sabre-dav) project, that would be great.

For an example of the functionality I am seeking to replace, please check out the Windows based eM Client software and how see how it fully integrates with the Fruux service

**Comment by hallamoore** [comment]
Another request: https://nylas.zendesk.com/agent/tickets/6627

**Comment by sebz1504** [comment]
+1 for carddav integration

**Comment by sebz1504** [comment]
+1

**Comment by jancborchardt** [comment]
Hi from the [Nextcloud](https://nextcloud.com) team :) we do a lot with CalDAV and CardDAV and it would also be cool to integrate on that front. People should indeed be able to simply sync their contacts. An input for their CardDAV server, username and token/password would be enough.

**Comment by ArchangeGabriel** [comment]
Regarding work around this in JS: http://stackoverflow.com/questions/14657473/caldav-in-node-js.

**Comment by hallamoore** [comment]
Another request: https://nylas.zendesk.com/agent/tickets/10512

**Comment by btnpushnmunky** [comment]
Any movement on the CalDAV front? I'm assuming this would help with Google Calendar integration?

**Comment by adueck** [comment]
I agree with @sysfu. It would be amazing to integrate [SabreDav](https://github.com/fruux/sabre-dav)/[fruux](https://fruux.com/)/[everything-else-it-runs-with](http://sabre.io/). I just got this e-mail from the [fruux.com](https://fruux.com) people after mentioning it to them:

>"If they do plan to add proper CardDAV/CalDAV support and have any questions, we are happy to help. Our open core (sabre/dav) also powers many other sync services (e.g. ownCloud, Posteo, etc.) so if they support us, they get support for many other services „for free“ and make even more of their users happy."

Plleeeeeaaassse bring this in.

**Comment by christophdb** [comment]
+1

**Comment by dougunder** [comment]
+1 
Recently setup carddav/caldav servers on my Synology.
Lack of support may force me to switch  :(

**Comment by jonathanmichel** [comment]
+1

**Comment by Timecrash** [comment]
+1

**Comment by enoch85** [comment]
Please stop with the +1. We all get notified.

Please use :+1: on the first post instead.

Thanks! :smiley:

**Comment by ArchangeGabriel** [comment]
Moreover, this project has been dead for years. The replacement is https://getmailspring.com/.
